LiteDB/Engine/Disk/StreamFactory/FileStreamFactory.cs

```diff
@@ -17,13 +17,15 @@ internal class FileStreamFactory : IStreamFactory
         private readonly string _password;
         private readonly bool _readonly;
         private readonly bool _hidden;
+		private readonly bool _useAesStream;
```

Comment:
please change the indentation to be space instead of tabs

Reply:
Done.